ElectricBass.h

```diff
@@ -0,0 +1,27 @@
+struct ElectricBass
```

Comment:
Where is the `#pragma once`?
Where is the `#include` statement for that JUCE_LEAK_DETECTOR macro?
- [ ] revise